adapters/mobfoxpb/mobfoxpb.go

```diff
@@ -50,7 +50,7 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.E
 	key, errKey := jsonparser.GetString(imp.Ext, "bidder", "key")
 	if errTag != nil && errKey != nil {
 		errs = append(errs, &errortypes.BadInput{
-			Message: fmt.Sprintf("Invalid or non existing key and tagId, atleast one should be present"),
+			Message: "Invalid or non existing key and tagId, atleast one should be present",
```

Comment:
Spacing Nitpick: `atleast` --> `at least`

Reply:
fixed in 6977017f together with several similar typos (below is grep from `v2.18.0` = `2c39db78b15d4fa42c4882d6be278c4716fba0cc`):
```bash
$ rg atleast
router/router.go
376:			// we might not have aliases defined, but will atleast show that the JSON file is parsable.

adapters/mobfoxpb/mobfoxpb.go
53:			Message: fmt.Sprintf("Invalid or non existing key and tagId, atleast one should be present"),

adapters/mobfoxpb/mobfoxpbtest/supplemental/bad-imp-ext.json
37:            "value": "Invalid or non existing key and tagId, atleast one should be present",

adapters/mobfoxpb/mobfoxpbtest/supplemental/imp_ext_string.json
33:            "value": "Invalid or non existing key and tagId, atleast one should be present",

adapters/mobfoxpb/mobfoxpbtest/supplemental/imp_ext_empty_object.json
33:            "value": "Invalid or non existing key and tagId, atleast one should be present",
```